### ISSUE TITLE
Accessibility - Accessibility page

### ISSUE THREAD

**Issue by Rhiana** [strategicdata/websurvey.com.au#7]
One of the standards of accessibility is including an accessibility page on the website which provides information on the accessibility features.

Here is an example of how it can be included from the [Business VIC](http://www.business.vic.gov.au/accessibility) which is a government site that is highly accessible to AA standard.

![screen shot 2015-12-01 at 12 45 19 pm](https://cloud.githubusercontent.com/assets/5127756/11490095/6b2997b2-9829-11e5-9555-01af012e8be3.png)

If we were keen to go into even more detail the [University of Melbourne](http://www.unimelb.edu.au/accessibility/index.html) goes even further, though perhaps a middle ground in our case would be more appropriate.

![screen shot 2015-12-01 at 12 51 58 pm](https://cloud.githubusercontent.com/assets/5127756/11490202/85760cf8-982a-11e5-9956-cac233a87748.png)
![screen shot 2015-12-01 at 12 51 48 pm](https://cloud.githubusercontent.com/assets/5127756/11490201/8275a644-982a-11e5-9faf-0f7d00be01af.png)

**Comment by adamc00** [comment]
The UoM example looks more like a resource for those producing content. I don't think that is what is intended by the requirement. I suspect that these locations are a better example of what is intended (they are linked from the above).

* http://library.unimelb.edu.au/discovery/accessibility
* https://lms.unimelb.edu.au/support/accessibility/

**Comment by Rhiana** [comment]
That makes sense, I think the first link is closer to what could work on our site, though I don't think we need any JAWS instructions on there. 
The compliance level, date, remaining issues (?)  and contact us if you find something.

**Comment by Rhiana** [comment]
I have added in a page and have had a go writing it up. 
If there is anything to change in the content (I'm fairly sure there will be) let em know and I'll update it.

I've put that we are working towards accessibility standards as we have not been formally assessed, so I'm also not sure if we can have the Conformance date part. Are we going to have it assessed?

![screen shot 2015-12-02 at 9 32 36 am](https://cloud.githubusercontent.com/assets/5127756/11516498/b1874e08-98d7-11e5-83f4-309c623fac56.png)

**Comment by michellenSD** [comment]
We do have some text related to the compliance of our surveys elsewhere on the site, so it probably makes sense just to reuse that:  "The majority of WebSurvey's standard functionality complies with the Web Content Accessibility Guidelines (WCAG) 2.0 guidelines at the AA standard. WebSurvey is able to provide fully compliant surveys where required. Please contact us for more information."

**Comment by adamc00** [comment]
Accessibility statements relate the the site they are hosted on so it should be about the accessibility of the websurvey.com.au site. The statement @michellenSD refers to belongs elsewhere regarding the creation of accessible surveys.

**Comment by michellenSD** [comment]
That makes sense.  Do we want to get the website assessed for compliance?  I wouldn't have thought so but I'm happy to be overruled.

**Comment by adamc00** [comment]
> Do we want to get the website assessed for compliance? I wouldn't have thought

Me neither.

**Comment by Rhiana** [comment]
No worries, it was more to figure out what to write on the page. I can update it so it relates to the website only and not the surveys.

**Comment by adamc00** [comment]
Perhaps we can have 2 headings "This website" and "Our surveys" the first is a standard accessibility statement about our site and the second is pretty much what Michelle wrote. We could have some more detail there about the process.

**Comment by michellenSD** [comment]
I think the current text on the accessibility page is fine @Rhiana - please just change the heading "Our Surveys Compliance Level" to "Our Surveys" and I think the site is ALMOST ready to go live!

**Comment by Rhiana** [comment]
Easy done [here](https://github.com/strategicdata/websurvey.com.au/commit/9311d2411ce2d18449ce3cc9b76c10ab6787a9eb)
